contracts/libs/0.8.x/MerkleLib.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+import "@openzeppelin-4.7/contracts/utils/cryptography/MerkleProof.sol";
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title Art Blocks Merkle Library
+ * @notice [TO FILL OUT]
+ * @author Art Blocks Inc.
+ */
+
+library MerkleLib {
+    using MerkleProof for bytes32[];
+    uint256 public constant DEFAULT_MAX_INVOCATIONS_PER_ADDRESS = 1;
+    bytes32 public constant CONFIG_MERKLE_ROOT = "merkleRoot";
+    bytes32 public constant CONFIG_USE_MAX_INVOCATIONS_PER_ADDRESS_OVERRIDE =
+        "useMaxMintsPerAddrOverride"; // shortened to fit in 32 bytes
+    bytes32 public constant CONFIG_MAX_INVOCATIONS_OVERRIDE =
+        "maxMintsPerAddrOverride"; // shortened to match format of previous key
```

Comment:
I think we probably want these to be internal, given that we intend to use this as a bundled internal library ya?